invenio_communities/views/communities.py

```diff
@@ -103,7 +103,8 @@ def communities_settings(community=None, logo=None, pid_value=None):
 
 @pass_community
 @require_community_owner
-def communities_settings_privileges(community=None, pid_value=None):
+@pass_community_logo
+def communities_settings_privileges(community=None, logo=None, pid_value=None):
```

Comment:
Minor: can you move it up, as for the other view func, so that the `require_community_owner` will be evaluated first (if I am not mistaken...)? I mean, we should check permissions first.

Can you please please change the 3 decorators implementation and have:
```
pid_value = kwargs['pid_value']
```
instead of
```
pid_value = kwargs.get('pid_value')
```
?